Detections/SecurityEvent/RDP_Nesting.yaml

```diff
@@ -0,0 +1,39 @@
+id: 69a45b05-71f5-45ca-8944-2e038747fb39
+name: RDP Nesting
+description: |
+  'Identifies when an RDP connection is made to a first system and then an RDP connection is made from the first system 
+  to another system with the same account within the same day.  
+  RDP connections are indicated by the logged EventID 4624 with LogonType = 10'
+severity: Medium
```

Comment:
Given there are quite a few legitimate use cases for this situation (granted same account name but still fairly common I would guess) should this be low rather than medium?

Reply:
Done